test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -10,47 +10,225 @@ Describe "New-PSSession basic test" -Tag @("CI") {
-            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue 
+            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue
             $scriptBlock = {Enter-PSSession -ComputerName Localhost -ConfigurationName JEA; Exit-PSSession}
-            & $scriptBlock
+            # Invoke the script block in a different PowerShell instance to that when TestDrive tries to delete $RoleCapDirectory,
+            # the transcription has finished and the files are not locked.
+            [powershell]::Create().AddScript($scriptBlock).Invoke()
             $headerFile = Get-ChildItem $transScriptFile | Sort-Object LastWriteTime | Select-Object -Last 1
             $header = Get-Content $headerFile | Out-String
-            $header | Should BeLike "Configuration Name: JEA"
+            $header | Should Match "Configuration Name: JEA"
         }
         finally
         {
             Unregister-PSSessionConfiguration -Name JEA -Force -ErrorAction SilentlyContinue
         }
     }
-    
-}
 
+}
 
 Describe "JEA session Get-Help test" -Tag @("CI", 'RequireAdminOnWindows') {
-    It "Get-Help should work in JEA sessions" -Pending {
+    BeforeAll {
+        Enable-PSRemoting -SkipNetworkProfileCheck
+    }
+
+    It "Get-Help should work in JEA sessions" {
         [string] $RoleCapDirectory = (New-Item -Path "$TestDrive\RoleCapability" -ItemType Directory -Force).FullName
         [string] $PSSessionConfigFile = "$RoleCapDirectory\TestConfig.pssc"
         try
         {
             New-PSSessionConfigurationFile -Path $PSSessionConfigFile -TranscriptDirectory $RoleCapDirectory -SessionType RestrictedRemoteServer
-            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue 
+            Register-PSSessionConfiguration -Name JEA -Path $PSSessionConfigFile -Force -ErrorAction SilentlyContinue
             $scriptBlock = {Enter-PSSession -ComputerName Localhost -ConfigurationName JEA; Get-Help Get-Command; Exit-PSSession}
-            $helpContent = & $scriptBlock
-            $helpContent | Should Not Be $null 
+            # Invoke the script block in a different PowerShell instance to that when TestDrive tries to delete $RoleCapDirectory,
+            # the transcription has finished and the files are not locked.
+            $helpContent = [powershell]::Create().AddScript($scriptBlock).Invoke()
+            $helpContent | Should Not Be $null
         }
         finally
         {
             Unregister-PSSessionConfiguration -Name JEA -Force -ErrorAction SilentlyContinue
             Remove-Item $RoleCapDirectory -Recurse -Force -ErrorAction SilentlyContinue
         }
     }
-    
-}
\ No newline at end of file
+}
+
+Describe "Remoting loopback tests" -Tags @('CI', 'RequireAdminOnWindows') {
```

Comment:
These finish in about 12 seconds. I think that should be OK. Thoughts?

Reply:
I am not sure but the testing guidelines say most CI tests take less than one second and the "Slow" tag should be considered if a test takes longer.  Since these are critical remoting functions it seems like we should leave them as CI tests, but probably should add the "Slow" tag.